readme.md

```diff
@@ -67,14 +67,14 @@ First, you'll run rollup to watch your `src/` module and automatically recompile
 
 ```bash
 npm start # runs rollup with watch flag
+npm link <your-module-name> # optional if using yarn
```

Comment:
`npm start` in this shell will keep running since it's watching for changes. So this shell won't be able to run `npm link`.

I think you want the `npm link` to be moved to right before line 77 `cd example`.

Reply:
Actually, scratch that, we should just remove the `npm link` suggestion from the readme entirely as we're already doing this step when `create-react-library` runs [here](https://github.com/transitive-bullshit/create-react-library/blob/master/lib/create-library.js#L99).

Great catch!

Reply:
Nvmd :)